packages/node_modules/@webex/internal-plugin-conversation/src/conversation.js

```diff
@@ -249,6 +249,43 @@ const Conversation = WebexPlugin.extend({
     return activity;
   },
 
+
+  /**
+  * Gets an array of activities with an array of activity URLS
+  * @param {Array} activties
+  * @param {String} cluster
+  * @returns {Promise<Object>} Resolves with the activities
+  * TODO: add cluster functionality when clusters are ready
+  */
+  bulk_activities_fetch(activties, cluster) {
```

Comment:
Can we change this to `bulkActivitiesFetch` to fit the naming of the rest of the methods in the plugin?

Reply:
yes sure will change that